.travis.yml

```diff
@@ -9,67 +9,44 @@ stages:
+      if: type = push and branch =~ /^(master|merge)/
+      script: make docker
     - name: Allen Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/allen_smFISH.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/allen_smFISH.py
     - name: MERFISH Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py
     - name: Assay Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/assay_comparison.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/assay_comparison.py
     - name: osmFISH Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/osmFISH.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/osmFISH.py
     - name: DARTFISH Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py
     - name: Simulation Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/Starfish_simulation.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/Starfish_simulation.py
     - name: ISS Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/ISS_Pipeline_-_Breast_-_1_FOV.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/ISS_Pipeline_-_Breast_-_1_FOV.py
     - name: Released Notebooks
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag = cron
-      script:
-        - make install-pypi
-        - make run_notebooks
+      if: type = cron and branch =~ /^(master|merge)/
+      script: make check_notebooks install-travis run_notebooks
+    - name: Check Requirements
+      if: type = cron AND branch = /^master$/
+      script: make refresh_all_requirements check_requirements
```

Comment:
This particular test will break all the time.  What's the expected frequency of this cron?

Reply:
At the moment, it's set to weekly.

Reply:
hm... let's see how it goes and we can tweak how often we want to run it. :)

Reply:
:) Agreed. Two options would be to make it a soft-fail so by visiting travis you can see it's red and needed, but the overall build would be green. Alternatively, the build could open the PR with the bump instead of just complaining. Or just remove this. Whatever turns out to be useful and least annoying.